helm/methode-image-set-mapper/values.yaml

```diff
@@ -8,8 +8,10 @@ replicaCount: 2
 image:
   repository: coco/methode-image-set-mapper
   pullPolicy: IfNotPresent
-#resources:
-#  limits:
-#    memory: 256Mi
+resources:
+  requests:
+    memory: 512Mi
+  limits:
+    memory: 1Gi
```

Comment:
Hm, I see there was a pick above 1Gi. Could we have a bit more than 1Gi to make sure the app won't be killed by OOM?

Reply:
True, I see it now. I'll adjust it a bit.